src/components/BTreeVisualizer.vue

```diff
@@ -162,6 +169,23 @@ export default {
       }
       this.currentFrame = this.currentSequence.frames.length - 1;
     },
+    reanimate() {
+      const frames = [];
+      this.currentSequence.frames.forEach((f) => frames.push(f));
+      this.currentSequence.frames = [frames[0]];
+      this.currentFrame = 0;
+      this.isAnimating = true;
+      for (let i = 1; i < frames.length; i += 1) {
+        setTimeout(() => {
+          this.currentSequence.addFrame(frames[i]);
+          if (i === frames.length - 1) {
+            this.isAnimating = false;
+          }
+          this.currentFrame += 1;
+          this.sequencesList = Object.assign(this.sequencesList);
+        }, i * 500);
```

Comment:
we should stop hardcoding this number and assign it to some value of the instance (we can take it up later to dynamically change the speed)

Reply:
also, was it possible to reuse the `addSequenceAsync` method?

Reply:
TODO